requirements.txt

```diff
@@ -1,5 +1,6 @@
 https://github.com/opencivicdata/pupa/archive/master.zip
 https://github.com/opencivicdata/python-legistar-scraper/archive/hcg/tweaks.zip
+https://github.com/datamade/scrapelib/archive/hcg/accept-response.zip
```

Comment:
Oh yes, this PR also brings in a change I made upstream that allows us to skip 410 requests without retries.